eox-core-v-1-0/prose/edit/src/schema-elements-01-definitions.md

```diff
@@ -27,3 +27,5 @@ The EoX Timestamp (`eox_timestamp_t`) type of value type `string` contains the t
 ```
 
 Therefore, it MUST either be a `string` with format `date-time` or use the constant value `tba`.
+
+OpenEoX Consumer MUST treat `tba` as a date greater than the maximum date.
```

Comment:
Best effort, I guess. I do not think we can write more or better. I hope we can find use cases spanning across updates of such `tba` containing instances to then see, if the ecosystem wants to support a version n with such a void and n+1 with a real date smaller (earlier) than the prior latest date.

Version n:

```yaml
$schema: 'https://docs.oasis-open.org/openeox/v1.0/schema/core.json'
end_of_life: '2035-11-19T15:59:59Z'
end_of_security_support: tba
last_updated: '2025-11-19T16:00:00Z'
```
and then n+1:

```yaml
$schema: 'https://docs.oasis-open.org/openeox/v1.0/schema/core.json'
end_of_life: '2035-11-19T15:59:59Z'
end_of_security_support: '2035-11-19T00:00:00Z'
last_updated: '2025-11-20T22:49:00Z'
```

Reply:
You brought up a valid point with your version n example: we need to ignore `tba` in certain test cases...